test/ILLink.RoslynAnalyzer.Tests/generated/ILLink.RoslynAnalyzer.Tests.Generator/ILLink.RoslynAnalyzer.Tests.TestCaseGenerator/Inheritance.Interfaces.StaticInterfaceMethodsTests.g.cs

```diff
@@ -8,7 +8,7 @@ public sealed partial class StaticInterfaceMethodsTests : LinkerTestBase
 	{
 
 		[Fact]
-		public Task BaseProvidesInterfaceMethod ()
+		public Task VarianceBasic ()
```

Comment:
Do we know why this happens? I do see that the VarianceBasic had a private Main method and is fine that we now have the Fact being generated. But why the BaseProvidesInterfaceMethod disappeared? And why there are no other tests? inside the Inheritance.Interfaces.StaticInterfaceMethods folder I see 10 files.